### ISSUE TITLE
Set timeout not correctly logged

### ISSUE THREAD

**Issue by MichielYKamp** [Omenia/robotframework-whitelibrary#86]
**Describe the bug**
When setting the timeout with keyword "set white find window timeout" the new value is used but not logged.

**To Reproduce**
Use the following keywords in a test:
```
set white find window timeout  10s
attach application by id  <id>
attach window  <non-existing window name>
```

**Expected behavior**
I expect to see the value that I had set in the report

**Actual behaviour**
Instead I see the default value of 30 seconds. See also the screenshot below.

**Screenshots**
In the screenshot you can also see that the logging of "set white find window timeout" is also off.

![image](https://user-images.githubusercontent.com/39919027/52120921-2ef20c80-261e-11e9-8e53-a8d76b3a6eb2.png)

**Comment by eeromakiesko** [comment]
The issue has been found in WhiteStack implementation. Incorrect log message will be corrected during exception handling improvements that will follow soon.

**Comment by eeter** [comment]
Fix is now available in the stable release (v1.3.0).
